numpy/lib/utils.py

```diff
@@ -13,9 +13,86 @@
 __all__ = [
     'issubclass_', 'issubsctype', 'issubdtype', 'deprecate',
     'deprecate_with_doc', 'get_include', 'info', 'source', 'who',
-    'lookfor', 'byte_bounds', 'safe_eval'
+    'lookfor', 'byte_bounds', 'safe_eval', 'show_runtime'
     ]
 
+
+def show_runtime():
+    """
+    Print information about various resources in the system
+    including available intrinsic support and BLAS/LAPACK library
+    in use
+
+    See Also
+    --------
+    show_config : Show libraries in the system on which NumPy was built.
+
+    Notes
+    -----
+    1. Information is derived with the help of `threadpoolctl`
```

Comment:
If you want this to be a link, you have to provide the url. Otherwise use double back-ticks.

Reply:
Thanks yeah it renders now: https://output.circle-artifacts.com/output/job/79c7902f-6c77-4b35-8f43-d55ceaee6ff5/artifacts/0/doc/build/html/reference/generated/numpy.show_runtime.html, all though it will give flake8 errors, I'm not sure why adding new line like this did not work: https://github.com/numpy/numpy/blob/17d730ae32f5f60c9c2ca75d202b4e866debd686/numpy/lib/npyio.py#L1846-L1847